snapcraft/plugins/v1/_plugin.py

```diff
@@ -86,6 +87,15 @@ def __init__(self, name, options, project=None):
         self.options = options
 
         if project:
+            if isinstance(project, Project) and project._get_build_base() not in (
+                "core",
+                "core16",
```

Comment:
Maybe this is a good time to drop "core16"?

Reply:
We could, but it is in more places than here.